Readme.md

```diff
@@ -97,13 +97,20 @@ Similar to Official AWS PHP SDK, AsyncAws provides waiters to let you wait
 until an long operation finished.
 
 ```php
-// create a queue Async and don't wait for the response.
-$sqsClient->createQueue(['QueueName' => 'fooBar']);
+use AsyncAws\DynamoDb\DynamoDbClient;
+
+$dbClient = new DynamoDbClient();
+$dbClient->createTable([]);
 
-$waiter = $sqsClient->queueExists(['QueueName' => 'fooBar']);
+// create a new table. It normally takes around 5 seconds to complete this action.
+$dbClient->createTable(['TableName' => 'foobar', /* ...*/]);
+
+$waiter = $dbClient->tableExists(['TableName' => 'foobar']);
 echo $waiter->isSuccess(); // false
+
 $waiter->wait();
 echo $waiter->isSuccess(); // true
+
```

Comment:
I use a DynamoDB example here. 

@GromNaN, please correct me if Im wrong. Calling `createTable()` will get a response within a second. But you cannot use the table yet because it is not actually created. That is why you need the waiter.

Reply:
Sorry, I forgot to answer. This is totally correct.